builtin/game/privileges.lua

```diff
@@ -58,4 +58,7 @@ core.register_privilege("zoom", {
 	description = "Can zoom the camera",
 	give_to_singleplayer = false,
 })
-
+core.register_privilege("debug", {
+	description = "Can enable various dangerous debug options or cheats",
```

Comment:
can you remove the "or cheats" part and replace it with "that may affect gameplay"?